S23/sandeep-1-kumar/LowerCasing.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Locale;
+public class LowerCasing {
+
+    public int upperCaseIrishA = 193;
+    public int upperCaseIrishE = 201;
+    public int upperCaseIrishI = 205;
+    public int upperCaseIrishO = 211;
+    public int upperCaseIrishU = 218; 
+
+    public String lowerCaseConversion(String languageType,String word)
+    {
+        String[] exceptedLanguages = {"zh" , "ja" , "th"};
+        Locale locale = Locale.forLanguageTag(languageType);
+        String language = locale.getLanguage();  
+        for(String element : exceptedLanguages)
+        {
+            if(element.equals(language))
+            {
+                return word;
+            }
+        }
+        String returningValue = word;
+        if(language.startsWith("tr") || language.startsWith("az"))
+        {   //->'/u0131' is a Unicode representation of the lowercase dotless I.
+             returningValue =word.replaceAll("[iI]", "\u0131");
```

Comment:
This doesn't look like the right behavior.  Did you add tests for all combinations of these characters?